src/arrays.js

```diff
@@ -1,61 +1,65 @@
 const getNthElement = (index, array) => {
-  // your code here
+index %= array.length;
+return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split (',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+return array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+ return strings.map(strings => strings.toUpperCase());
```

Comment:
2 spaces of indentation here not one :) 
Also, pay attention to any errors vscode shows you, often it will explain how to fix it for you. In this case it says "Strings is already declared in the upper scope". This is because youre using the same variable name in the map function which is a bad practice, you should call this something different. 'string' would be a good choice as its not taken and it represents the singular form of what you are looping over, so it reads better too
```
  return strings.map(strings => strings.toUpperCase());
```